apps/web/tsconfig.server.json

```diff
@@ -7,5 +7,5 @@
     "tsBuildInfoFile": "../../tmp/buildcache/apps/web/server",
     "types": ["node"]
   },
-  "include": ["server.ts"]
+  "include": ["server.ts"] //"include": ["server.ts", "**/*.js", "**/*.ts", "**/*.tsx", "next-env.d.ts"]
```

Comment:
Do we want this comment to be here?

Reply:
This def shouldn't be there.
This is test code to ensure that the web project throws on strict errors since the tsconfig.server.json include block overrides the base tsconfig of the web project

Reply:
overdue cleanup - fixed